backend/run.sh

```diff
@@ -14,11 +14,13 @@ for package in desispec desiutil; do
 	export PYTHONPATH=$QLF_ROOT/$package/py:$PYTHONPATH
 done
 
+export PYTHONPATH=$QLF_ROOT/framework/bin:$PYTHONPATH
+
 cd $QLF_ROOT
 echo "Initializing QLF Daemon..."
 
 # Start QLF daemon
-python -Wi framework/bin/qlf_daemon.py &> $QLF_ROOT/qlf_daemon.log &
+python -Wi framework/bin/servers.py &
```

Comment:
Are we not going to log the daemon anymore?

Reply:
I'm going to edit it. It's only a test.